tests/tests/datm_control_iau_gefs

```diff
@@ -0,0 +1,15 @@
+#
+#  DATM_MOM6_CICE_CMEPS_GEFS test
+#
+
+export TEST_DESCR="DATM-MOM6-CICE-CMEPS_GEFS - IAU"
+
+export CNTL_DIR="datm_control_gefs"
```

Comment:
If you turn on MOM_IAU, do you expect this test to reproduce the datm_control_gefs?

Reply:
No,  do I need to change CNTL_DIR to datm_control_iau_gefs?

Reply:
Yes, the CNTL_DIR is the test you are comparing against.